cmd/go.mod

```diff
@@ -1,10 +1,12 @@
 module github.com/awslabs/soci-snapshotter/cmd
 
-go 1.21
+go 1.22
+
+toolchain go1.22.6
```

Comment:
Go 1.22 min update should be handled by seperate PR.

Reply:
We should also remove the toolchain command, unless we have a reason to want 1.22.6

Reply:
Rebased and resolved conflicts for toolchain update.